resources/usr/bin/openhab

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
+
+## This is a utility script to simplify the use of openHAB 2
+## Intended to be used with a packaged install (.deb/.rpm)
+## License information available at /usr/share/openhab2/LICENSE.txt by default
+
+usage() {
+  echo "Usage:  openhab command [options]"
+  echo ""
+  echo "Possible commands:"
+  echo "    start [--debug]             -- Starts openHAB in the terminal."
+  echo "    stop                        -- Stops any running instance of openHAB."
+  echo "    status                      -- Checks to see if openHAB is running."
+  echo "    console                     -- Opens the openHAB console."
+  echo "    backup [filename]           -- Stores the current configuration of openHAB."
+  echo "    restore filename            -- Restores the state of openHAB to a backup."
+  echo "    showlogs                    -- Displays the log messages of openHAB as they appear."
+  echo ""
+  exit 2
+}
+
+# Script requires options, list them if none given
+if [ "$#" -eq "0" ] || [ "$1" = "-h" ] || [ "$1" = "--help" ] ; then
+  usage
+fi
+
+# Some functions require root
+checkRoot() {
+  if [ "$(id -u)" -ne 0 ]; then
+    optionName="$1"
+    echo "This option needs to run as root! (e.g. use sudo openhab $optionName ..)" >&2
```

Comment:
`echo "This option needs to run as root! (e.g. use 'sudo openhab $optionName')" >&2`
?